run.py

```diff
@@ -283,6 +284,11 @@ def draw_info_bb_selected(tmp_img):
 image_list = glob.glob(img_dir +'*.jpg')
 image_list.extend(glob.glob(img_dir + '*.jpeg'))
 #print(image_list)
+if args.sort:
+    image_list.sort()
+else:
+    np.random.seed(123)
```

Comment:
The seed is to ensure the random order (which might be wanted) is consistent between sessions. However, there's no guarantee glob will collect filepaths consistently, and so I'll need to sort the images before shuffling.